### PR TITLE
solution of py-feed-animals/2

### PR DESCRIPTION
**Pull request by vkleshko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,44 @@
...
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
```
Comment (ArsenPidhoretskyi) [inline app/main.py:20]: else: not needed. Just  do return
```suggestion
        return 0
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,44 @@
...
+
+    def bring_slippers(self) -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
+    total_sum_of_appetite = 0
+    for animal in animals:
+        total_sum_of_appetite += animal.feed()
+    return total_sum_of_appetite
```
Comment (ArsenPidhoretskyi) [inline app/main.py:44]: Use sum function

File: app/main.py [hunk 3]
```diff
@@ -1 +1,44 @@
...
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        super().__init__(name, appetite=7, is_hungry=is_hungry)
+
+    def bring_slippers(self) -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animals: list) -> int:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:40]: list of what? Provide inner type

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Comment by vkleshko** [comment]
i fixed all mistakes

**Pull request closed without merging**